Quantum-Control-Applications/Superconducting/Single Flux Tunable Transmon/SSB mixer for readout/IQ_blobs.py

```diff
@@ -23,41 +23,15 @@
 
 with program() as singleshot:
     n = declare(int)  # Averaging index
-    counter = declare(int)  # Number of tries for active reset
-
     Ig_st = declare_stream()
     Qg_st = declare_stream()
     Ie_st = declare_stream()
     Qe_st = declare_stream()
 
     with for_(n, 0, n < n_shot, n + 1):
-        # Reset the number of tries
-        assign(counter, 0)
         # Measure the g and e states
         I_g, Q_g, I_e, Q_e = ge_singleshot_measurement(cooldown_time)
```

Comment:
I'm not sure I like this macro...
Why don't we have the code in here? You've simply taken the entire script and moved it into a macro, which is only used once in a single file?

Reply:
WHen I was thee, it was actually quite convenient to test the different active reset protocols because it separates well the single_shot measurement of g and e from the active reset part and I used it as e demonstration of QUA macros. 
Right now I don't have a clear use-case in mind, so if you really hate it I don't mind removing it

Reply:
I wouldn't say I hate it, I just don't really see any benefit in having it.
It would make more sense to have something like:
```
active_reset (or change into a cooldown)
I_g, Q_g, = readout_macro()
active_reset (or change into a cooldown)
play('pi', 'qubit')
I_e, Q_e, = readout_macro()
```
This would use existing macros and would be more readable and customizable. 
WDYT?

Reply:
Yes I agree